### PR TITLE
Add test case for feature-1

### PR DESCRIPTION
**Pull request by alvarogitmmn**

### CODE REVIEW

File: TEST-FEATURE-1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Step 1: Login to system
+Step 2: Go to settings
```
Comment (alvarogitmmn) [inline TEST-FEATURE-1.md:2]: Missing corner case to validate
